icepyx/core/query.py

```diff
@@ -813,22 +832,6 @@ def product_all_info(self):
             self._about_product = is2ref.about_product(self._prod)
         pprint.pprint(self._about_product)
 
-    def latest_version(self):
```

Comment:
Running through the notebooks and we now have this error to address:

`AttributeError: 'Query' object has no attribute 'latest_version'`

Reply:
I don't see anywhere that Query calls latest_version. Do you recall what line caused this error?